src/Projectionist.php

```diff
@@ -135,6 +136,40 @@ public function getReactors(): Collection
         return $this->reactors->all();
     }
 
+    public function addEventHandler($eventHandlerClass)
```

Comment:
Can we simplify `addProjector` and `addReactor` to use this method under the hood?

Reply:
Like discussed on Slack, will leave it like it is.